client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-hosting-details-list.tsx

```diff
@@ -1,67 +1,47 @@
-import { trendingUp, next } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
-import { defaultHostingDetails } from '../constants';
-import wordpressCwvtechReportJson from '../wordpress-cwvtech-report.json';
+import cwvtechReportJson from '../cwvtech-report.json';
+import { useDefaultHostingDetails } from '../use-default-hosting-details';
 import { useGetUpgradePlanSiteMetrics } from './use-get-upgrade-plan-site-metrics';
 
 export const useUpgradePlanHostingDetailsList = () => {
 	const translate = useTranslate();
 	const { lcpPercentageDifference, fidPercentageDifference, isFetching } =
 		useGetUpgradePlanSiteMetrics();
 
-	let newDefaultHostingDetails = [ ...defaultHostingDetails ];
-	const newHostingDetails = [];
+	const newHostingDetails = useDefaultHostingDetails();
 
 	// LCP hosting details.
 	if ( lcpPercentageDifference > 0 ) {
-		const wordpressLCP = Math.floor( 100 * wordpressCwvtechReportJson.goodLCP );
-
-		const updatedHostingSpeedDetails = {
-			title: translate( 'Higher speed' ),
-			description: translate(
-				'%(wordpressLcpPercentage)s of sites on WordPress.com are at least %(sitePercentageDifference)s faster than yours.',
-				{
-					args: {
-						wordpressLcpPercentage: `${ wordpressLCP }%`,
-						sitePercentageDifference: `${ lcpPercentageDifference }%`,
-					},
-				}
-			),
-			icon: trendingUp,
-		};
-		newDefaultHostingDetails = newDefaultHostingDetails.filter(
-			( { id } ) => id !== 'increased-speed'
+		const goodLCPercentage = Math.round( 100 * cwvtechReportJson[ 'WordPress.com' ].goodLCP );
```

Comment:
Before, it was `Math.floor` to avoid saying that we have 100% of goodLCP when it was 99.9...%. But after updating the JSON it's less than that, so I changed it back to `Math.round`.

Reply:
Reverted since the new numbers were wrong. See https://github.com/Automattic/wp-calypso/pull/90966#discussion_r1613652549